src/Dfe.Spi.GraphQlApi.Application/GraphTypes/Inputs/AggregationRequestCondition.cs

```diff
@@ -0,0 +1,33 @@
+using Dfe.Spi.Common.Models;
+using Dfe.Spi.GraphQlApi.Application.GraphTypes.Enums;
+using GraphQL.Types;
+
+namespace Dfe.Spi.GraphQlApi.Application.GraphTypes.Inputs
+{
+    public class AggregationRequestConditionModel
+    {
+        public string Field { get; set; }
+        public DataOperator Operator { get; set; }
+        public string Value { get; set; }
+    }
```

Comment:
This model exists in the Common Models lib. You could swap it out with no issues.